packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -236,7 +236,7 @@ export class LocalPersistentCache implements IPersistedCache {
     async removeAllEntriesForDocId(docId: string): Promise<void> {
         Array.from(this.cache)
         .filter(([key]) => {
```

Comment:
you can simplify: 
.filter(([key]) => key.startsWith(docId));

Reply:
done